share/availability/window.go

```diff
@@ -7,8 +7,10 @@ import (
 )
 
 const (
-	RequestWindow = 30 * 24 * time.Hour
-	StorageWindow = RequestWindow + time.Hour
+	SamplingWindow = 7 * 24 * time.Hour // Ref: CIP-036
+	RequestWindow  = SamplingWindow
+	// StorageWindow TODO @renaynay: point it back to RequestWindow + 1 hr in following release
```

Comment:
Nit: more like the opposite - request window will need to be storage window minus 1 hour after network migration is done.

Reply:
Can we leave it til migration is done? I can open issue